src/plots/cartesian/set_convert.js

```diff
@@ -360,7 +361,6 @@ module.exports = function setConvert(ax) {
             Lib.notifier(
                 'Something went wrong with axis scaling',
                 'long');
-            ax._gd._replotting = false;
             throw new Error('axis scaling');
```

Comment:
:hocho: `ax._gd._replotting = false;` on axis scaling errors. Isn't that error throw enough?